test/dlc_tests/ec2/test_efa.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import pytest
+
+from test.test_utils import CONTAINER_TESTS_PREFIX, LOGGER, is_tf2
+from test.test_utils.ec2 import get_ec2_instance_type
+
+EFA_SCRIPT = os.path.join(CONTAINER_TESTS_PREFIX, "test_efa_sanity.py")
+
+EFA_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p3dn.24xlarge", processor="gpu")
+
+
+@pytest.mark.parametrize("ec2_instance_type", EFA_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_sanity(training, ec2_connection, region, gpu_only, py3_only):
+    run_efa_test(training, ec2_connection, region, docker_executable="nvidia-docker", container_name="efa")
```

Comment:
Why is it necessary to make `docker_executable` a parameter that is set to `docker` by default?

Isn't this test meant to be run on GPU instances only?

Reply:
Ya. fair point. will set it to nvidia-docker